src/assets/data.json

```diff
@@ -0,0 +1,491 @@
+[
+  {
+    "name": "FakeCorp",
```

Comment:
can you use chat gpt or gemini to get some fake names, descriptions ... so we have different data